.github/workflows/test.yml

```diff
@@ -26,6 +26,8 @@ jobs:
         with:
           perl-version: ${{ matrix.perl-version }}
 
-      - run: cpanm -L local --installdeps .
+      - run: cpanm -L local --notest --installdeps .
 
-      - run: prove -Ilocal/lib/perl5 -Ilib -lv t
+      - run: prove -lv t
+        env:
+          PERL5OPT: -Ilocal/lib/perl5 -Ilib -MTest2::Plugin::GitHub::Actions::AnnotateFailedTest
```

Comment:
>`-MTest2::Plugin::GitHub::Actions::AnnotateFailedTest`

It's very useful since we don't have to add `use Test2::Plugin::GitHub::Actions::AnnotateFailedTest;` to all tests!

I think it may be good to describe this `PERL5OPT` setting in SYNOPSIS as a usage of this plugin.